pkgs/by-name/in/interactive-html-bom/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  fetchFromGitHub,
+  kicad-small,
+  lib,
+  python3Packages,
+}:
```

Comment:
```suggestion
{
  lib,
  python3Packages,
  fetchFromGitHub,
  kicad-small,
}:
```

Reply:
It is already to just sort this alphabetically.